Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessagesCountDecorationView.swift

```diff
@@ -5,8 +5,8 @@
 import StreamChat
 import UIKit
 
-/// The date separator view that groups messages from the same day.
-open class ChatThreadRepliesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
+/// The view used to show a separator when there are unread messages.
+open class ChatMessagesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
```

Comment:
I can see some issues with this view:
1. The doc comment is incorrect since it is also used for Thread replies.
2. We should not have logic in the names of the views. What I mean is that a customer might customize this and not even use the "count" of the messages. I've seen customers who just want to have a "New Messages" banner instead of displaying the messages count.
3. Having the content as a String does not provide context on what the content actually is. It also means that if you want to change the content, the customer will need to customize more than 1 view